network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/PhaseTapChangerImpl.java

```diff
@@ -106,8 +106,7 @@ public PhaseTapChangerStep getCurrentStep() {
 
     @Override
     public void remove() {
-        //TODO
-        throw new UnsupportedOperationException("TODO");
+        parent.setPhaseTapChanger(null);
```

Comment:
no need to update resource ?

Reply:
I rebased the code on this PR #340 , the update resource is in the implementation

Reply:
Ah yes I see